client/css/ProductOverviewCSS.scss

```diff
@@ -0,0 +1,94 @@
+.productOverviewContainer {
+  display: flex;
+}
+
+.productInfo{
+  border: 2px solid black;
+  width: 50%;
+  margin: 0 1%;
+}
+
+.imageGalleryContainer {
+  display: flex;
+  height: 1000px;
+  width: 100%;
+  align-items: center;
+}
+
+.mainImageContainer {
+  display: flex;
+  height: 100%;
+  width: 100%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  border: 2px solid black;
+  justify-content: space-around;
+  align-items: center;
+  order: 3;
+};
+
+
+.mainImage {
+  width: 50%;
+  min-height: 50%;
+  max-height: 50%;
+}
+
+.productDescription {
+  border: 2px solid black;
+  margin: 1% 0;
+}
+
+.arrowsThumbnailContainer {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  border: 2px solid black;
+  order: 1;
+  width: 15%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.thumbnailContainer {
+  display: flex;
+  flex-direction: column;
+  height: 90%;
+  border: 2px solid black;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+}
+
+.thumbnail {
+  height: 100px;
+  width: 100px;
+  margin: 5% 0;
+
+}
+
+.leftArrow {
+  background-color: rgba(0, 0, 0, 0.35);
+  order: 2;
+  left: 102px;
+  cursor: pointer;
+}
+
+.rightArrow {
+  background-color: rgba(0, 0, 0, 0.35);
+  order: 4;
+  cursor: pointer;
+}
+
+#selected {
+  border: 3px solid blue;
+}
+
+.upArrow {
```

Comment:
can we combine these selectors since they're doing the same thing?

Reply:
Combined